README.md

```diff
@@ -12,7 +12,9 @@
 ### Basics
 
 ```
-make help
+# fresh install
+ansible-playbook --inventory=inventory --ask-pass bootstrap.yml --user=root
```

Comment:
I don't think running playbooks as `root` is a good idea. This hides away the fact that the tasks should be performed with elevated privileges inside the playbook.

It also implies a big security issue - it **should not** be possible to log in as root over SSH.

Reply:
Good catch, there will be a lot of refactoring so this is the right PR